include/pisa/query/queries.hpp

```diff
@@ -92,12 +92,13 @@ struct Query {
 
 [[nodiscard]] std::function<void(const std::string)> compute_parse_query_function(
```

Comment:
In the last PR it was pending to simplify the name of this function to `compute_query_parser`, `resolve_query_parser` or similar. Maybe this bug fix could be used to improve this too.